setuptools/command/build_py.py

```diff
@@ -11,7 +11,7 @@
 
 try:
     from setuptools.lib2to3_ex import Mixin2to3
-except ImportError:
+except Exception:
```

Comment:
I think it's important to avoid catching broad exception classes because it may shadow future bugs that would be hard to debug. Besides, adding non-ImportError exceptions here would require a huge code comment explaining the motivation.

Reply:
@webknjaz what about just `EOFError`?

Reply:
Given that disutils is going to disappear from stdlib, it's probably reasonable to have ImportError too. Although I'm not entirely sure where the fix should be applied to so I'll leave that to @jaraco to decide.

Reply:
Sorry, I mean just `ImportError, EOFError`

Reply:
Yeah, I get what you mean. I just have a feeling that something is off but I can't tell what exactly.

Reply:
How about a PR that vendors fissix?